packages/rspack/src/config/types.ts

```diff
@@ -338,7 +338,7 @@ export interface ModuleOptionsNormalized {
 export type AvailableTarget =
 	| "node"
 	| "web"
```

Comment:
```suggestion
type AvailableTarget =
	| "node"
	| "web"
```

Will intellisense still works if remove the `export`?

Reply:
The reason for exporting here is that it is used in other file 
https://github.com/modern-js-dev/rspack/blob/4a3bb3eff8b468c69056f23285f72c30377f4842/packages/rspack/src/config/defaults.ts#L44